src/main/java/org/apache/commons/codec/language/bm/Rule.java

```diff
@@ -488,11 +496,12 @@ private static RPattern pattern(final String regex) {
         final boolean endsWith = regex.endsWith("$");
         final String content = regex.substring(startsWith ? 1 : 0, endsWith ? regex.length() - 1 : regex.length());
         final boolean boxes = content.contains("[");
+        final int contentLength = content.length();
 
         if (!boxes) {
             if (startsWith && endsWith) {
                 // exact match
-                if (content.length() == 0) {
+                if (contentLength == 0) {
```

Comment:
Just use `String#isEmpty()` for 0-length checks on Strings.

Reply:
agree. fixit